test/test_Transformers.cc

```diff
@@ -421,6 +422,197 @@ TEST_F(Transformer_New_Test, test_NodeEliminator_SecondEdgeUsed) {
     EXPECT_EQ(Validator(logic).validate(originalGraph, translatedResult), Validator::Result::VALIDATED);
 }
 
+TEST_F(Transformer_New_Test, test_SimpleNodeEliminator_HyperEdge_Safe) {
+    Options options;
+    options.addOption(Options::LOGIC, "QF_LIA");
+    options.addOption(Options::COMPUTE_WITNESS, "true");
```

Comment:
The plot shows the results without witnesses on LIA-nonlin.
I can run a quick sanity check with witnesses turned on.

LIA-Lin benchmarks are unaffected by this change, since there are no hyperedges there.

Reply:
Ok, then on smth with hyperedges, exactly to check that hyperedges backward translation doesn't brake anything

Reply:
I ran all LIA-nonlin benchmarks from CHC-COMP-22 with a timeout of 20 seconds and all witnesses were successfully validated.